frontend/apps/reader/modules/readersearch.lua

```diff
@@ -102,7 +101,7 @@ function ReaderSearch:onShowFulltextSearchInput()
     }
 
    -- checkboxes
-    self.check_button_regex = self.check_button_regex or CheckButton:new{
+    self.check_button_regex = CheckButton:new{
```

Comment:
Just asking, why did you remove this?
(It is a small optimisation, but it looked a bit ugly.)

Reply:
With it in the ceckbuttons did not get checked from time to time (more precisely the were not refreshed sometimes.)